### ISSUE TITLE
Multi-definition issue of including the DPCTL header in multiple CPP sources

### ISSUE THREAD

**Issue by chengjunlu** [IntelPython/dpctl#946]
Hi,

I have two CPP sources include the DPCTL header.
The multi-definition issue happen in linking:
```
/usr/bin/ld: /tmp/Stream-e835c3.o: in function `std::__detail::_Hash_node<std::pair<_object const* const, std::vector<_object*, std::allocator<_object*> > >, false>::_M_next() const':
/home/guangyey/CLionProjects/pytorch/torch/include/pybind11/pybind11.h:662: multiple definition of `import_dpctl()'; /tmp/Module-08230d.o:/home/guangyey/CLionProjects/pytorch/torch/include/pybind11/pybind11.h:662: first defined here
```

I can work around the issue by change the `import_dpctl` to static.

**Comment by diptorupd** [comment]
@chengjunlu Current master should already have the fix. Can you try the latest development snapshot package? The package will need to be installed via Conda.

Refer https://github.com/IntelPython/dpctl#installing-the-bleeding-edge

**Comment by chengjunlu** [comment]
@diptorupd Thanks for the suggestion.
I met the issue on the latest master trunk commit ab46d49c1476d53964e5fa77533cfd8ffacba1ed.

**Comment by oleksandr-pavlyk** [comment]
@chengjunlu I am suggesting you build dpctl from sources at the present tip of the master at 36c8f711c28cf2873bf71d177122150d061648c3. The problem you are reporting should be fixed there.

**Comment by diptorupd** [comment]
ping @chengjunlu. Does current tip of master work for you? Once you let us know we can close the ticket.

**Comment by chengjunlu** [comment]
@diptorupd 
Confirmed the issue has been fixed on the latest master.
Please close this JIRA.

**Comment by oleksandr-pavlyk** [comment]
Closing as fixed by #941
